.env

```diff
@@ -0,0 +1 @@
+TRANSIFEX_TOKEN="1/e5ce3430adf6f018626a6c269b2ef582b4348aed"
```

Comment:
do not commit the env file, that should be a local thing per user. git igore it

Reply:
Good catch, yeah.